runtime/compiler/p/codegen/ForceRecompilationSnippet.cpp

```diff
@@ -47,6 +47,9 @@ uint8_t *TR::PPCForceRecompilationSnippet::emitSnippetBody()
    uint8_t             *buffer = cg()->getBinaryBufferCursor();
    TR::SymbolReference  *induceRecompilationSymRef = cg()->symRefTab()->findOrCreateRuntimeHelper(TR_PPCinduceRecompilation);
    intptr_t startPC = (intptr_t)((uint8_t*)cg()->getCodeStart());
+   // The relo runtime expects this kind of address to be zero in a relocatable binary, since it uses
+   // |= to update the address chunks. See TR_PPC64RelocationTarget::storeAddressSequence.
+   intptr_t startPCRegValue = cg()->comp()->compileRelocatableCode() ? 0 : startPC;
```

Comment:
Actually, I think that EDO is still disabled when JITServer is enabled, but maybe that restriction can be lifted with this PR as well.

Reply:
startPC will need to be relocated because the code is generated at the server and installed in the code cache at the client.
At the moment we disable EDO when compiling remotely because of the relocation issue.
I am not familiar with `TR_RamMethodSequence`

Reply:
The doc says: "`TR_RamMethodSequence`|Relocates the J9Method pointer of the method being compiled based on the sequence enum specified.|", so it's not the startPC but rather the J9Method pointer

Reply:
Would that need to be relocated whenever the startPC would be? I can always add a new query if none of the existing ones are precise enough.